projects/storefrontlib/src/lib/my-account/order/order-details/order-details.component.ts

```diff
@@ -20,7 +20,7 @@ export class OrderDetailsComponent implements OnInit, OnDestroy {
     private routingService: RoutingService
   ) {}
 
-  order$: Observable<any>;
+  order$: Observable<Order>;
```

Comment:
Can we add types on lines: 46 (`address`), 59 (`billingAddress`), 74 (`payment`) 86 (`shipping`)?